public/coffee/main/event.coffee

```diff
@@ -1,6 +1,39 @@
 define [
 	"base"
 ], (App) ->
+	CACHE_KEY = "countlyEvents"
+
+	_getEventCache = () -> 
+		eventCacheStr = window.localStorage.getItem CACHE_KEY
```

Comment:
we have modules/localStorage which wraps this up a little bit prevents some errors (try catch)